src/main/java/com/cloudant/client/internal/views/PageMetadata.java

```diff
@@ -30,23 +43,79 @@
     /**
      * The parameters needed to request the page described by this metadata
      */
+    @Expose
+    @SerializedName("v")
     ViewQueryParameters<K, V> pageRequestParameters;
 
     /**
      * The page number of the page to retrieve
      */
+    @Expose
+    @SerializedName("n")
     long pageNumber;
     /**
      * Indicate the paging direction for this metadata
      */
+    @Expose
+    @SerializedName("d")
     PagingDirection direction;
 
     enum PagingDirection {
+        @SerializedName("F")
         FORWARD,
+        @SerializedName("B")
         BACKWARD
     }
 
     PageMetadata(PagingDirection direction) {
         this.direction = direction;
     }
+
+    static <K, V> PageMetadata<K, V> fromString(final ViewQueryParameters<K, V> initialParameters,
```

Comment:
I don't think this name really conveys what the method is doing, particularly given you're merging with the initialParameters in addition to using the string.

It's somewhat complicated that the pagination parameter doesn't include some of the ViewQueryParamters, which makes calling it "parameter" even more complicated -- I now have many types of parameter in my head.